src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class StorageDaoImplTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+
+    @BeforeAll
+    public static void setupStorage() {
```

Comment:
Each test should not depend on the results of the previous one. Thus, we will not be able to clean the storage, but we need to do this after each test.
Add to storage is only necessary in methods that check for the existence of users.

Reply:
I don't really understand the point about adding to storage being only necessary in specific methods. By the way, I understand, that each test method should be independant from the others, so i tried to achieve that in my current tests setting up the users **before** all the test methods